util/options.py

```diff
@@ -29,6 +29,9 @@ def get(opt, default=None):
 def get_platform() -> str:
     return opts.get("platform", "n64")
 
+def get_endianess() -> str:
```

Comment:
This is more a function of the options than an option itself..I tend to think of stuff in here as user-configurable, but this isn't, at least the way it's currently written. but it does seem like a good place for it...hmm. not sure

Reply:
yeah, I think that works!